pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -218,8 +217,6 @@ type TidbClusterSpec struct {
 	// cluster component is needed to reload the configuration change.
 	// UpdateStrategyRollingUpdate will create a new ConfigMap with the new configuration and rolling-update the
 	// related components to use the new ConfigMap, that is, the new configuration will be applied automatically.
-	// +kubebuilder:validation:Enum=InPlace;RollingUpdate
```

Comment:
The validation can be reserved?

Reply:
The `+kubebuilder:default=IfNotPresent` for ImagePullPolicy should also be removed.